etc/config/cpp_for_opencl.amazon.properties

```diff
@@ -176,49 +176,53 @@ compiler.armv8-full-cpp4oclclang-trunk.semver=(trunk allfeats)
 compiler.armv8-full-cpp4oclclang-trunk.options=-target aarch64-none-linux-gnu --gcc-toolchain=/opt/compiler-explorer/arm64/gcc-12.2.0/aarch64-unknown-linux-gnu --sysroot=/opt/compiler-explorer/arm64/gcc-12.2.0/aarch64-unknown-linux-gnu/aarch64-unknown-linux-gnu/sysroot -march=armv8.8-a+crypto+profile+rng+memtag+sve2+sve2-bitperm+sve2-sm4+sve2-aes+sve2-sha3+tme+brbe+f32mm+f64mm+fp16fml+ls64+sme+sme-f64f64+sme-i16i64+sme2
 
 # SPIR-V Assembly
-group.armcpp4oclclang32spir.groupName=Arm 32-bit clang (SPIR-V asm)
-group.armcpp4oclclang32spir.compilers=armv7-cpp4oclclang-trunk-spir:armv7-cpp4oclclang-trunk-assertions-spir
-group.armcpp4oclclang32spir.isSemVer=true
-group.armcpp4oclclang32spir.compilerType=spirv
-group.armcpp4oclclang32spir.supportsExecute=false
-group.armcpp4oclclang32spir.instructionSet=arm32
-# The -Dkernel= -D__kernel= workaround is required to prevent the Clang crash reported in https://llvm.org/PR50841
```

Comment:
As far as I can see, your PR fixing this hasn't yet been merged, right? (LLVM's issue tracker is a bit hard to read, so sorry if I missed something) Do we not need the workaround anymore?

Reply:
@AnastasiaStulova thanks again for your contribution: we'd love to merge. Can you answer @RubenRBS 's question so we know where we are on this?

Reply:
Sorry for delay, this clang bug is only for compiling to Arm targets. When we compile for SPIR/SPIR-W: `-triple spir`, `-triple spir64`, `-triple spirv32` or `-triple spirv64` no workaround is needed as clang just compiles correctly.